app/values/hls_manifest.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class HlsManifest
+  def self.for(file_set:, file_metadata:, as: nil, auth_token: nil)
+    return unless file_metadata.mime_type.first.to_s == "application/x-mpegURL"
```

Comment:
`return unless file_metadata.hls_manifest?`?

Reply:
Or should we change the name of that method to `hls_file?`?

Reply:
Thank you!